backend/src/Controller/Image/ImageController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller\Image;
+
+use App\Controller\BaseController;
+use App\Service\Image\ImageService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+
+/**
+ * @Route("v1/image/")
+ */
+class ImageController extends BaseController
+{
+    private ImageService $imageService;
+
+    public function __construct(SerializerInterface $serializer, ImageService $imageService)
+    {
+        parent::__construct($serializer);
+        $this->imageService = $imageService;
+    }
+
+    /**
+     * Get images. THIS JUST FOR TESTING CREATE IMAGE FUNCTIONS
```

Comment:
@isGranted is missing

Reply:
As I mentioned in the doc, I create the API for testing, so that I didn't granted it for specific user.